calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Start to extract the background model from radar and lidar data.                         |
+| `add_lidar_radar_pair`     | `std_srvs::srv::Empty`                                | Add lidar-radar pairs for calibration.                                                   |
+| `delete_lidar_radar_pair`  | `std_srvs::srv::Empty`                                | Delete the previous lidar-radar pair.                                                    |
+| `send_calibration`         | `std_srvs::srv::Empty`                                | Send the calibration result to the sensor calibration manager.                           |
+
+## Parameters
+
+### Core Parameters
+
+| Name                                        | Type          | Default Value                                           | Description                                                                                                                                                        |
+| ------------------------------------------- | ------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `radar_parallel_frame`                      | `std::string` | `base_link`                                             | The frame that the radar frame optimizes the transformation to.                                                                                                    |
+| `msg_type`                                  | `std::string` | `radar tracks`/`radar scan`                             | The type of the input radar objects. (Not available yet, currently only support radar tracks)                                                                      |
+| `transformation_type`                       | `std::string` | `yaw_only_rotation_2d` `svd_2d` `svd_3d` `roll_zero_3d` | The algorithms for optimizing the transformation between radar frame and radar parallel frame. (Not available yet)                                                 |
```

Comment:
It is a string, to `algorithms` is not correct

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/15afdf233595d82b2b3ad4a675d06293f7d3bdb3

Reply:
It is a variable from a set of multiple options. However, the parameter is one, not several of them.

Reply:
add a new explanation in https://github.com/tier4/CalibrationTools/pull/164/commits/8e4f0de76716f68c310f88abb87b88bf8921649b

Reply:
The second to last period is not correct in this case, but should be fixed in your next PR anyways